docs/unit-test-case.md

```
@@ -1,21 +1,147 @@
+|--------|-----:|----:|----:|----:|----:|----:|----:|----:|
+| Notes 2.0 | ×1.0 | — | — | — | — | 15 | 6 | 21 |
+| Light 2 | ×1.0 | — | — | — | — | 10 | 4 | 14 |
+| Light 3 | ×1.1 | — | — | 2 | — | 33 | 11 | 46 |
+| Light 4 | ×1.2 | — | — | 46 | — | — | — | 46 |
+| Light 5 | ×1.3 | — | — | 47 | 2 | — | — | 49 |
+| Light 6 | ×1.5 | 76 | 9 | 9 | 1 | — | — | 95 |
+| Chorus Light 6 | ×3.0 | 21 | 14 | 47 | 11 | 49 | 15 | 157 |
+| **合計** | — | **97** | **23** | **151** | **14** | **107** | **36** | **428** |
+
+
+##　スコア計算の概要
```

markdownlint-cli2 (0.22.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)